clj/build.boot

```diff
@@ -51,6 +62,16 @@
            (install)
            (target)))
 
+(deftask build-cljs
+  "build clojurescript"
+  []
+  (comp
+   (test-cljs)
+   (cljs :optimizations :advanced)))
+
+(deftask build []
+  (comp (build-cljs) (build-clj)))
```

Comment:
But as I understand we need `:exit? true` to exit if there are failed tests.  Simple workaround would be just to change the order of the tasks.